src/devices/Bh1750fvi/Bh1750fvi.cs

```diff
@@ -45,15 +45,15 @@ public double LightTransmittance
         /// <summary>
         /// Creates a new instance of the BH1750FVI
         /// </summary>
-        /// <param name="sensor">I2C Device, like UnixI2cDevice or Windows10I2cDevice</param>
+        /// <param name="i2cDevice">>The I2C device used for communication.</param>
```

Comment:
ditto

Reply:
Hmmmm.  I must of copied paste something common.  Lemme check